### DISCUSSION TITLE
Compiling Stellarium on Kubuntu 22.10

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#2936] [Q&A]
When I do in ~temp/stellarium directory on Kubuntu 22.10 command

$ git fetch https://github.com/Stellarium/stellarium

It updates some source files. And then I do in directory ~/temp/stellarium/build/unic

$ cmake -DCMAKE_INSTALL_PREFIX=/opt/stellarium ../.. 

And then

$ make -j10

It compiles stellarium executable file to directory ~/temp/stellarium/build/unix/source

And when I start it, it displays Stellarium 1.1 version.

Before about two weeks ago the compiled verdion didn't contain all translated phrases.
I think that this way of compilation doesn't get latest source files. How to get them?
For example today vedrsion 1.2 files. What I do wrong?

I like to compile Stellarium on this computer, because it starts more quickly than
downloaded AppImage. And online quories dialog works without crash.

Btw, I sent a message to Reddit to Stellar group about Stellarium version 1.2 release.
It got 54 likes. :)

**Comment by 10110111** [comment]
`git fetch` doesn't update your local branch, only the index. You need `git pull` instead.

**Reply by ilkant** [reply]
Thanx! Compiled version is now 1.2+ and it works. Even the online query dialog works. My git experience is not very good.
